.github/kokoro/steps/rapidwright.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
I assume that the RapidWright change will be merged separately before this one?

Reply:
Ya, #1673

Reply:
Base RapidWright change is now merged.